core/store/src/main/java/uk/gov/gchq/gaffer/store/serialiser/lengthvalue/EntityIdSerialiser.java

```diff
@@ -78,4 +78,9 @@ public EntityId deserialiseEmpty() throws SerialisationException {
     public boolean preservesObjectOrdering() {
         return false;
```

Comment:
Please could you change this to:

return null != vertexSerialiser && vertexSerialiser.preservesObjectOrdering();